src/components/Popups/ReplyField/withMentionDecorator.tsx

```diff
@@ -0,0 +1,24 @@
+import { CompositeDecorator, ContentBlock, ContentState, EditorState } from 'draft-js';
```

Comment:
This file probably move to a more common location, like `src/components`.

Reply:
> This file probably move to a more common location, like `src/components`.

But it's only used in the `ReplyField`. Also `MentionItem` has to move to `src/components` with it, which may be confusing to people.